### DISCUSSION TITLE
无法使用vcpkg下载东西

### DISCUSSION THREAD

**Discussion by woshiluziyi** [microsoft/vcpkg#36795] [Q&A]
我用的pycharm

<img width="484" alt="9c9e34c717658f5bd95a2427e7867ea" src="https://github.com/microsoft/vcpkg/assets/156077448/46f0d73d-8025-4e03-82e3-2fdff244d0cc">

**Comment by wkingnet** [comment]
大哥，pycharm是开发python的，vcpkg是C++语言用的。
